public/editor/scripts/ui/publisher.js

```diff
@@ -14,6 +14,7 @@ var TEXT_UNPUBLISH = strings.get("publishDeleteBtn");
 var TEXT_UNPUBLISHING = strings.get("publishUnpublishingIndicator");
 var TEXT_UPDATE_PUBLISH = strings.get("publishChangesBtn");
 var TEXT_UNPUBLISH_WARNING = strings.get("publishDeleteWarning");
+var TEXT_PUBLISHED_HEADER = strings.get("publishHeaderOnline");
```

Comment:
let's call the `TEXT_PUBLISH_HEADER_ONLINE`
and add another variable here called `TEXT_PUBLISH_HEADER` that stores the `publishHeader` string